lib/capybara/node/base.rb

```diff
@@ -2,6 +2,14 @@
 
 module Capybara
   module Node
+    # @!macro waiting_behavior
+    #   If the driver is capable of executing JavaScript, this method will wait for a set amount of time
+    #   and continuously retry finding the element until either the element is found or the time
+    #   expires. The length of time this method will wait is controlled through {Capybara.configure default_max_wait_time}.
+    #
+    #   @option options [false, true, Numeric] wait
```

Comment:
I have a question. The `default_max_wait_time` option defined in `Capybara.configure` can take a `Numeric` value, but on the other hand, the option here can take `false`, `true`, or `Numeric`. Is this correct? 🤔 

https://github.com/teamcapybara/capybara/blob/87e79c2daf75035e6d5d22b43ba25cc9a7cd9920/lib/capybara.rb#L78

Reply:
true/false are for overriding behaviors around waiting, they are not valid values for default_max_wait_time - false ends up being equivalent to `wait: 0` while true can locally override things like the setting of `predicates_wait`

Reply:
Thanks. I see. 👍